### ISSUE TITLE
The test results are not collect

### ISSUE THREAD

**Issue by koalakodai** [jgkwak95/AU-GAN#14]
@jgkwak95 
I am running **train.py** and **test.py** on **512x256** pix images.
Each is executed as follows.

`python main.py --dataset_dir bdd100k --phase train--experiment_name bdd_exp --batch_size 1 --load_size 258 --fine_size 256`

`python main.py --dataset_dir bdd100k --phase test --experiment_name bdd_exp --batch_size 1 --load_size 258 --fine_size 256 --use_uncertainty True`

I am facing the following problems.

① When I am running --phase train
 loss function does not decrease as shown in the picture. 
(It's always been **nan**.)

![image](https://github.com/jgkwak95/AU-GAN/assets/131735967/e10a3ebf-0832-437c-9d70-e817b40f71e1)

② When I am running --phase test
 the fake_img is not displayed as shown in the picture. 
 
![image](https://github.com/jgkwak95/AU-GAN/assets/131735967/7895c713-0d30-4db6-9804-ffc6d1a7add9)

I believe ① and ② are related.
If you have a solution, I would like to know.
I'm looking forward to hearing from you soon.
